presto-native-execution/presto_cpp/main/PrestoExchangeSource.h

```diff
@@ -250,6 +250,15 @@ class PrestoExchangeSource : public velox::exec::ExchangeSource {
   const std::string host_;
   const uint16_t port_;
   const folly::SSLContextPtr sslContext_;
+  // If true, we directly build serialized presto page from iobufs allocated by
+  // proxygen in http response.
+  const bool noBufferCopy_;
```

Comment:
Can we name it enableBufferCopy_ (negate to be more intuitive) and below one rename it to immediateBufferCopy_
as they are tightly associated.

Reply:
Agree with @tanjialiang. noBufferCopy is non-intuitive.